docs/upgrade/3.9.0.rst

```diff
@@ -1,4 +1,4 @@
-.. _upgrade-to-3.9.0:
+drop-down.. _upgrade-to-3.9.0:
```

Comment:
This is another one that looks odd to me. Not sure the context for drop-down here.

Reply:
Thank you, Mark. This is an error on my part. I erased that. I will be more careful.